flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -37,15 +37,19 @@ def read_dataset(dataset_info: Union[DatasetMeta, Text, int],
                  read_dataset_processor=None,
                  name: Text = None) -> Channel:
     """
-    Read dataset from the dataset operator. It can read dataset from external system.
-
-    :param dataset_info: Information about the dataset which will be read. Its type can be DatasetMeta
-                         of py:class:`ai_flow.meta.dataset_meta.DatasetMeta` or Text or int. The dataset_info
-                         means name in the metadata service when its type is Text and it means id when its type is int.
-                         The ai flow will get the dataset from metadata service by name or id.
-    :param read_dataset_processor: The python user defined function in read dataset operator. User can write their own logic here.
-    :param name: Name of the read_dataset operator.
-    :return: Channel: data output channel.
+    Reads the dataset through the user-defined processor based on the metadata of dataset. The processor reads dataset
+    from external system and generates the :class:`~ai_flow.meta.ai_graph.ai_node.ReadDatasetNode` with
+    :class:`~ai_flow.meta.dataset_meta.DatasetMeta`.
+
+    :param dataset_info: Information about the dataset read. The type of dataset information could be
+                                      :class:`~ai_flow.meta.dataset_meta.DatasetMeta` or Text or int. The `dataset_info`
+                                      means the name of dataset when the type is Text and it means the id of dataset
+                                      when the type is int, which gets the metadata of dataset by the id.or name of
+                                      dataset at default.
```

Comment:
Current description is fine but I think it can be more concise like this:

`or Text(i.e. the name of dataset) or int(i.e. the id of dataset). If the type is Text or int, this method will get the metadata of dataset by the name or id of dataset by default.`